presto-main/src/test/resources/catalog.json

```diff
@@ -0,0 +1,40 @@
+{
+  "catalogs": [
+    {
+      "user": "admin",
+      "catalog": ".*",
+      "allow": true
+    },
+    {
+      "catalog": "secret",
+      "allow": false
+    },
+    {
+      "user": ".*",
+      "catalog": "open-to-all",
+      "allow": true
+    },
+    {
+      "catalog": "all-allowed",
+      "allow": true
+    },
+    {
+      "user": "alice",
+      "catalog": "alice-catalog",
+      "allow": true
+    },
+    {
+      "user": "bob",
```

Comment:
It isn't, but I wanted to test this particular use case, when `user` and `catalog` are mentioned, and `allow` is false.